crates/dwallet-mpc-centralized-party/src/lib.rs

```diff
@@ -434,7 +435,14 @@ pub fn verify_secp_signature_inner(
     let hashed_message =
         message_digest(&message, &hash_type.try_into()?).context("Message digest failed")?;
     let (r, s): (secp256k1::Scalar, secp256k1::Scalar) = bcs::from_bytes(&signature)?;
```

Comment:
this is no longer the signature, look at the code please. Use `ECDSASecp256k1Signature`